modules/Bio/EnsEMBL/Variation/Pipeline/BaseVariationProcess.pm

```diff
@@ -149,5 +149,28 @@ sub run_cmd {
   }
 }
 
+sub get_assembly {
+  my $self = shift;
+  my $core_dba = $self->get_species_adaptor('core');
+  my $dbc = $core_dba->dbc;
+  my $current_db_name = $dbc->dbname();
+  my $species = $self->param('species');
+  my $species_id = $self->get_species_id($dbc, $current_db_name, $species);
+  my $sth = $dbc->prepare("SELECT version FROM ".$current_db_name.".coord_system WHERE species_id = ".$species_id." ORDER BY rank LIMIT 1;");
+  $sth->execute();
+  my $assembly;
+  $sth->bind_columns(\$assembly);
+  $sth->execute();
+  $sth->fetch();
+  $sth->finish();
+
+  return $assembly;
+}
+
+sub get_species_id {
```

Comment:
Bio::EnsEMBL::DBSQL::MetaContainer method species_id can be used to get species_id